files/fluentd/run.sh

```diff
@@ -15,49 +12,20 @@ if [ -n "$sniffer" -a -f "$sniffer" ] ; then
     fluentdargs="$fluentdargs -r $sniffer"
 fi
 
-if [[ $VERBOSE ]]; then
-  set -ex
-  fluentdargs="$fluentdargs -vv --log-event-verbose"
-  echo ">>>>>> ENVIRONMENT VARS <<<<<"
-  env | sort
-  echo ">>>>>>>>>>>>><<<<<<<<<<<<<<<<"
-else
-  set -e
-  fluentdargs="-q --suppress-config-dump $fluentdargs"
-fi
-
+set -e
+fluentdargs="-q --suppress-config-dump $fluentdargs"
 
 issue_deprecation_warnings() {
     : # none at the moment
 }
 
-if [ -z "${JOURNAL_SOURCE:-}" ] ; then
-    if [ -d /var/log/journal ] ; then
-        export JOURNAL_SOURCE=/var/log/journal
-    else
-        export JOURNAL_SOURCE=/run/log/journal
```

Comment:
what was the case that it would be `/run/log/journal` again?

Reply:
pre/post atomic? :man_shrugging:   I removed it because we explicitly rely on one path with the generator so this block is of no value unless we are going to modify the generator